packages/@lwc/engine/src/framework/vm.ts

```diff
@@ -89,6 +89,7 @@ export interface UninitializedVM {
     cmpProps: any;
     cmpSlots: SlotSet;
     cmpTrack: any;
+    cmpFields: any;
```

Comment:
don't add a new one... use `cmpTrack` instead because in another PR (wire reform), I'm renaming this to `cmpFields` as a single place to store such values.